rudder-agent/SOURCES/Makefile

```diff
@@ -276,10 +277,14 @@ rudder.8.gz:
 	$(WGET) http://www.rudder-project.org/rudder-doc-${RUDDER_MAJOR_VERSION}/rudder.8
 	gzip -f rudder.8
 
-initial-promises: ./rudder-sources
+initial-promises: ./rudder-sources initial-ncf
 	rm -rf ./initial-promises/
 	cp -r ./rudder-sources/rudder-techniques/initial-promises/node-server/ ./initial-promises
 
+initial-ncf: ./rudder-sources
+	rm -rf ./initial-ncf
+	cp -r ./rudder-sources/ncf/tree ./initial-ncf
```

Comment:
For initial-ncf to work, we'll need an initial ncf.conf file too, that should contain the configuration to use log_rudder, and the 5309 CFEngine port (and whatever else we change in there).